consensus/types/src/signed_execution_payload_envelope.rs

```diff
@@ -0,0 +1,109 @@
+use crate::test_utils::TestRandom;
+use crate::*;
+use educe::Educe;
+use serde::de::{Deserializer, Error as _};
+use serde::{Deserialize, Serialize};
+use ssz_derive::{Decode, Encode};
+use superstruct::superstruct;
+use test_random_derive::TestRandom;
+use tree_hash_derive::TreeHash;
+
+#[superstruct(
+    variants(Gloas, NextFork),
```

Comment:
Don't superstruct ahead of time

Reply:
fixed in [794fcb3](https://github.com/sigp/lighthouse/pull/7923/commits/794fcb37649ab39c16a79e204baf533d1e820cec)